main.go

```diff
@@ -136,6 +137,13 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "OcmAgent")
 		os.Exit(1)
 	}
+	//if err = (&fleetnotification.ManagedFleetNotificationReconciler{
```

Comment:
Is this commented out for a reason?

Reply:
this should be uncommented when the controller implemented in OSD-15391, currently it is just dummy code.